cms/djangoapps/contentstore/views/component.py

```diff
@@ -30,6 +30,11 @@
 from django.utils.translation import ugettext as _
 from models.settings.course_grading import CourseGradingModel
 
+try:
+    from xblock_django.models import XBlockDeprecatedAdvancedComponentConfig
+except ImportError:
+    XBlockDeprecatedAdvancedComponentConfig = None
```

Comment:
You don't need to worry about an `ImportError`. `xblock_django` will always be available.